### ISSUE TITLE
Missing method OnImpulse (Type: FrooxEngine.ProtoFlux.Runtimes.Execution.Nodes.ContinuationRelay)

### ISSUE THREAD

**Issue by troyBORG** [Yellow-Dog-Man/Resonite-Issues#2590]
### Describe the bug?

I noticed in my Headless Log showing up a bunch
```Missing method OnImpulse (System.Action on Element: ID5F92200, Type: FrooxEngine.ProtoFlux.Runtimes.Execution.Nodes.ContinuationRelay, World: Avatar Station (3.0 BETA) - Avatars and Tutorials for New Users, IsRemoved: False, IsDestroyed: False, IsDisposed: False, Enabled: True
```

I know you've been working on "Type Management system and the .NET 8.0 headless"

I wonder if this one got missed?

### To Reproduce

Open `resrec:///G-United-Space-Force-N/R-e02565ca-b2aa-431c-82bd-9c7670713df8` on .net headless
check log file

### Expected behavior

Not to see a bunch of missing method errors?

### Resonite Version Number

2024.7.15.1359

### What Platforms does this occur on?

Linux

### What headset if any do you use?

Headless

### Log Files

[2024.7.15.1359-2024-07-16-00_56_57.log](https://github.com/user-attachments/files/16244182/2024.7.15.1359-2024-07-16-00_56_57.log)

**Comment by Frooxius** [comment]
No, this has nothing to do with the Type Management system. 

This looks like leftovers from some legacy content, referencing things that do not exist.

Are you experiencing any specific breakage?

**Comment by troyBORG** [comment]
No, it didn't seem like it was affecting anything. Originally, I thought it might have been related to my other [issue](https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/2589), but the world that seemed like it was kicking off a bunch of those events, if I didn't have it run, the problem is unrelated.
